src/main/resources/application.yml

```diff
@@ -8,6 +8,12 @@ spring:
   codec:
     max-in-memory-size: 10MB
 
+  security:
```

Comment:
This appears to be the magic sauce that was missing. The env var value will be pulled in at runtime from the various helm values files (above)